v4/export/ir_impl.go

```diff
@@ -241,9 +249,9 @@ func splitTableDataIntoChunks(
 	chunkIndex := 0
 	nullValueCondition := fmt.Sprintf("`%s` IS NULL OR ", escapeString(field))
 LOOP:
-	for cutoff <= max {
+	for max.Cmp(cutoff) >= 0 {
 		chunkIndex += 1
-		where := fmt.Sprintf("%s(`%s` >= %d AND `%s` < %d)", nullValueCondition, escapeString(field), cutoff, escapeString(field), cutoff+estimatedStep)
+		where := fmt.Sprintf("%s(`%s` >= %d AND `%s` < %d)", nullValueCondition, escapeString(field), cutoff, escapeString(field), cutoff.Add(cutoff,new(big.Int).SetUint64(estimatedStep)))
```

Comment:
`new(big.Int).SetUint64(estimatedStep)` will be used several times. We can use a variable to store this value.